apps/test/unit/code-studio/components/teacher-homepage/CourseCardTest.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+import  CourseCard from '@cdo/apps/templates/teacherHomepage/CourseCard';
+
+const NONE_ENROLLED = {
+  courseName: "CSP Unit 2 - Digital Information",
+  description: "Explore how more complex digital information is represented and manipulated through computation and visualization",
+  image: "this is where there will be the source for the photo",
+  link: "link to wherever",
+  assignedSections: []
+};
+
+const ENROLLED_ONE = {
+   ...NONE_ENROLLED,
+   assignedSections: ["Section 1"]
+ };
+
+const ENROLLED_MANY = {
+  ...NONE_ENROLLED,
+  assignedSections: ["Section 1", "Section 2", "Section 3"]
+};
+
+describe('CourseCard component tests', () => {
+  let renderer;
+
+  beforeEach(() => {
+    renderer = ReactTestUtils.createRenderer();
```

Comment:
We've been favoring enzyme over ReactTestUtils for newer stuff (you can see an example here https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/templates/progress/ProgressLessonTest.js).